File: C/2020/12限/while2.c
//  while2.c
//  cl  while2.c
#include <stdio.h>

int main() {
    int i = 0;
    do {
        printf("%d \n", i);
    } while(i ++ < 10);

    do {
        printf("false do-while\n");
    } while(0);

    return 0;
}

File: GameServer/2025/Sat12/while2.php
<?php  // while2.php

function hit() {
    $r = random_int(1, 6) + random_int(1, 6);
    return $r;
}

// $r = hit();
// var_dump($r);

// 「10以上なら命中」で、命中するまで攻撃する
while(hit() < 10) {
    echo "はずれた... <br>";
}
echo "命中した!!";

